cmd/cstor-pool-mgmt/pool/pool_test.go

```diff
@@ -569,8 +609,87 @@ func TestCheckValidPool(t *testing.T) {
 				Status: apis.CStorPoolStatus{},
 			},
 		},
+		"Valid-StripedDisks1": {
+			expectedError: nil,
+			test: &apis.CStorPool{
+				TypeMeta: v1.TypeMeta{},
+				ObjectMeta: v1.ObjectMeta{
+					UID: types.UID("abc"),
+				},
+				Spec: apis.CStorPoolSpec{
+					Disks: apis.DiskAttr{
+						DiskList: []string{"/dev/sdb", "/dev/sdc", "/dev/sdd"},
+					},
+					PoolSpec: apis.CStorPoolAttr{
+						CacheFile:        "/tmp/pool1.cache",
+						PoolType:         "striped",
+						OverProvisioning: false,
+					},
+				},
+				Status: apis.CStorPoolStatus{},
+			},
+		},
+		"Valid-StripedDisks2": {
+			expectedError: nil,
+			test: &apis.CStorPool{
+				TypeMeta: v1.TypeMeta{},
+				ObjectMeta: v1.ObjectMeta{
+					UID: types.UID("abc"),
+				},
+				Spec: apis.CStorPoolSpec{
+					Disks: apis.DiskAttr{
+						DiskList: []string{"/dev/sdb"},
+					},
+					PoolSpec: apis.CStorPoolAttr{
+						CacheFile:        "/tmp/pool1.cache",
+						PoolType:         "striped",
+						OverProvisioning: false,
+					},
+				},
+				Status: apis.CStorPoolStatus{},
+			},
+		},
+		"Valid-StripedDisksEmptyPoolType": {
+			expectedError: nil,
+			test: &apis.CStorPool{
+				TypeMeta: v1.TypeMeta{},
+				ObjectMeta: v1.ObjectMeta{
+					UID: types.UID("abc"),
+				},
+				Spec: apis.CStorPoolSpec{
+					Disks: apis.DiskAttr{
+						DiskList: []string{"/dev/sdb", "/dev/sdc", "/dev/sdd"},
+					},
+					PoolSpec: apis.CStorPoolAttr{
+						CacheFile:        "/tmp/pool1.cache",
+						PoolType:         "",
+						OverProvisioning: false,
+					},
+				},
+				Status: apis.CStorPoolStatus{},
+			},
+		},
```

Comment:
This test case should fail. We do not want to default empty to striped I guess.
Empty pool type is invalid.

Reply:
Thanks @sonasingh46 for the info, I will remove this.